src/Compiler/Service/ServiceParsedInputOps.fs

```diff
@@ -1661,7 +1661,8 @@ module ParsedInput =
             | SynType.Array (_, t, _)
             | SynType.HashConstraint (t, _)
             | SynType.MeasurePower (t, _, _)
-            | SynType.Paren (t, _) -> walkType t
+            | SynType.Paren (t, _)
```

Comment:
Please also adjust 

* traverseSynType in ServiceParseTreeWalk
* walkType in `GetEntityKind`  (what is that funciton anyway?  It has so much code walking duplication!)

Maybe also remove the default `| _ ->`  pattern matching in those two cases so these missing cases would have been discovered, and check for other missing cases there

Reply:
Thanks, I've addressed the missing cases.